vassal-app/src/main/java/VASSAL/counters/SendToLocation.java

```diff
@@ -379,6 +380,12 @@ public Command myKeyEvent(KeyStroke stroke) {
         final ChangeTracker tracker = new ChangeTracker(this);
         setProperty(BACK_MAP, getMap());
         setProperty(BACK_POINT, getPosition());
+
+        // Mark moved and generate movement trail if compatibility pref turned on
+        if (Boolean.TRUE == GameModule.getGameModule().getPrefs().getValue(GlobalOptions.SEND_TO_LOCATION_MOVEMENT_TRAILS)) {
```

Comment:
Should use `.equals`, not `==` for these.

Reply:
Oops. Fixed.